lib/t/feeds.ex

```diff
@@ -633,8 +633,181 @@ defmodule T.Feeds do
     |> Repo.delete_all()
   end
 
+  # Since Live schedule
+
+  def live_schedule do
+    [
+      {_mon = 1, :newbie, [newbies_live_start_at(), newbies_live_end_at()]},
+      {_tue = 2, :newbie, [newbies_live_start_at(), newbies_live_end_at()]},
+      {_wed = 3, :newbie, [newbies_live_start_at(), newbies_live_end_at()]},
+      {_thu = 4, :real, [_start = ~T[19:00:00], _end = ~T[21:00:00]]},
+      {_fri = 5, :newbie, [newbies_live_start_at(), newbies_live_end_at()]},
+      {_sat = 6, :real, [_start = ~T[18:00:00], _end = ~T[20:00:00]]},
+      {_sun = 7, :newbie, [newbies_live_start_at(), newbies_live_end_at()]}
+    ]
+  end
```

Comment:
This would be the place to configure them.

Reply:
yeah